### DISCUSSION TITLE
The scroll page position does not persist when using wire:navigate

### DISCUSSION THREAD

**Discussion by nunowar** [livewire/livewire#7454] [7. Bugs (archived)] [answered]
### Livewire version

3.2.6

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://wirebox.app/b/47rre

### Steps To Reproduce

Check the link: https://wirebox.app/b/47rre and scroll down until you reach the menu and click on menu item. The scroll position will be reset and it's not supposed to be.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by CasEbb** [comment]
Can confirm this is bugged. Problem seems to be located in [js/plugins/navigate/scroll.js](https://github.com/livewire/livewire/blob/main/js/plugins/navigate/scroll.js). `document.body.scrollTop` is always 0. `document.body` should probably be replaced with `document.documentElement`.

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and I believe this has been fixed in PR #9341 which will be available in the next release, so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
